_config.yml

```diff
@@ -8,6 +8,15 @@ description: > # this means to ignore newlines until "baseurl:"
 
 twitter_username: jekyllrb
 github_username:  jekyll
+rss: rss
+dribbble_username: jekyll
+facebook_username: jekyll
+flickr_username: jekyll
+instagram_username: jekyll
+linkedin_username: jekyll
+pinterest_username: jekyll
+youtube_username: jekyll
+googleplus_username: +jekyll
 
```

Comment:
I would rather only keep existing Jekyll accounts and leave other services commented to avoid unnecessary 404s.